### ISSUE TITLE
Firefox, Headless chrome issue, The character encoding of the plain text document was not declared. 

### ISSUE THREAD

**Issue by prudhvidandamudi** [DevExpress/testcafe#5028]
#4547 # What is your Test Scenario?

 I am testing our companies application using testcafe and firefox
### What is the current behavior?
We are using sso login, and I am using a login and logout logic for each test.
at the time of logout, after clicking the logout button, the browser just displays a blank screen and stays there for 2 -3 mins and then it closes and opens the browser to continue on the next test. if there are many tests, the execution is taking forever.

when I check the console log output, it's showing the following error
 "**The character encoding of the plain text document was not declared. The document will render with garbled text in some browser configurations if the document contains characters from outside the US-ASCII range. The character encoding of the file needs to be declared in the transfer protocol or file needs to use a byte order mark as an encoding signature.**"

I guess this also happens when I am running tests via headless chrome, as execution time taking longer than normal.

### What is the Expected behavior?
Expecting to log out and continue with the next test without 3 min wait delay
 
### What is your web application and your TestCafe test code?

 

<details>
<summary>Your complete configuration file (if any):</summary>

```
{
    "screenshots": {
        "path": "automation_suite/artifacts/screenshots",
        "fullPage": false,
        "pathPattern": "${FIXTURE}/${TEST}/${FILE_INDEX}.png",
        "takeOnFails": true
    },
    "videoPath": "automation_suite/artifacts/videos",
    "videoOptions": {
        "singleFile": false,
        "failedOnly": true,
        "pathPattern": "${FIXTURE}/${TEST}/${FILE_INDEX}.mp4"
    },
    "tsConfigPath": "tsconfig.json",
    "reporter": [
        {
            "name": "spec"
        },
        {
            "name": "html",
            "output": "automation_suite/artifacts/reports/report.html"
        }
    ],
    "speed": 1,
    "selectorTimeout": 15000,
    "skipJsErrors": true,
    "concurrency": 1
}

```
</details>

<details>
<summary>Your complete test report:</summary>

```

```
</details>

<details>
<summary>Screenshots:</summary>

![image](https://user-images.githubusercontent.com/46937269/80557010-78f1ab80-89a3-11ea-896c-d02111f377f5.png)

![image](https://user-images.githubusercontent.com/46937269/80557017-7ee78c80-89a3-11ea-971c-a93cbd235768.png)
</details>

 
### Your Environment details:
 
* testcafe version:                   1.8.4
* node.js version:                    v10.16.3

* browser name and version:  Firefox 74.0.1
* platform and version:          Windows 10

**Comment by miherlosev** [comment]
Hi @prudhvidandamudi

We have not faced such issues before. Could you please share an example in which we can reproduce the issue locally?

**Comment by prudhvidandamudi** [comment]
> Hi @prudhvidandamudi
> 
> We have not faced such issues before. Could you please share an example in which we can reproduce the issue locally?

I did some debugging when it's happening 
when I click on the logout button the application is routing to https:// url where the above mentioned scenario is happening. Instead of clicking the Logout button, I tried to use t.navigate method to navigate to the logout url (this time in the URL I used http://) then it's completely working fine. So maybe this should have something to do with https:// and http://
I hope this helps.
